src/sentry/models/organizationmember.py

```diff
@@ -594,3 +595,15 @@ def is_only_owner(self) -> bool:
             .exists()
         )
         return is_only_owner
+
+
+def organization_member_post_delete(instance: OrganizationMember, **kwargs):
+    region_outbox = None
+    with transaction.atomic():
+        region_outbox = instance.outbox_for_update()
+        region_outbox.save()
```

Comment:
Is there a way we could generate the outbox message when the record is deleted? In a post_delete signal we could have the member removed but fail to save the outbox message as they are in separate transactions.

Reply:
@markstory I had missed that this exists on the `OrganizationMember` model: https://github.com/getsentry/sentry/blob/d50cfed8fbf59a959620b3452c0ce6e0114d51c8/src/sentry/models/organizationmember.py#L226-L229

I've revised the pull request accordingly.